### DISCUSSION TITLE
[TOOL] How to unbind all instance cooldown to player?

### DISCUSSION THREAD

**Discussion by dev-master21** [azerothcore/azerothcore-wotlk#3909] [Tools]
How i can to reset all instance cooldown to all players (online\offline players) without reboot server?

command .in un all just reset cooldown to selected player.

**Comment by ghost** [comment]
You've tried this ? .instance unbind 

Syntax: .instance unbind <mapid|all> [difficulty]
  Clear all/some of player's binds

**Comment by ghost** [comment]
Ah yes I didn't understand the : .in un all
What you want is a command or a module?

Because if this command doesn't correspond to your request I don't think you can find a better one at the moment.

**Comment by BarbzYHOOL** [comment]
You can do it for 1 player, but not for all players with a command.

Maybe delete in the DB but I don't recommend doing it if you don't know what you're doing
